articles/app-service/includes/quickstart-php/quickstart-php-azure-portal-pivot.md

```diff
@@ -0,0 +1,200 @@
+
+ ![On the Create Web App screen, on the Basics tab, in the Project Details section, for Resource group, select the 'Create new' link and add a group named 'myResourceGroup'.](../../media/quickstart-php/azure-portal-create-app-service-create-resource-group.png)
+
+4. Under **Instance details**, set the following settings:
+   - **Name**: Type a globally unique name for your web app. 
+   - **Publish**: Select *Code*.
+   - **Runtime stack**: Select *PHP 8.0*. 
+   - **Operating system**: Select *Linux*.
+   - **Region**: Select the region you want to serve your app from.
+
+ ![On the Create Web App screen, on the Basics tab, in the Instance Details section, use a globally unique name. Select the options for Publish, Runtime stack, Operating system, and region.](../../media/quickstart-php/azure-portal-create-app-service-instance-details-php8.png)
+
+5. Under **App Service Plan**, select **Create new** App Service Plan. Type *myAppServicePlan* for the name. Then select **OK**.
+
+ ![On the Create Web App screen, on the Basics tab, in the App Service Plan section, select the 'Create new' link. Enter myAppServicePlan in the textbox.](../../media/quickstart-php/azure-portal-create-app-service-create-app-service-plan.png)   
+
+6.  To change to the Free tier, next to **Sku and size**, select **Change size**. 
+   
+7.  In the Spec Picker, select **Dev/Test** tab, select **F1**, and select the **Apply** button at the bottom of the page.
+
+ ![In the Spec Picker, select Dev/Test, then select F1. Finally select the Apply.](../../media/quickstart-php/azure-portal-create-app-service-select-free-tier.png)   
+
+    
+8. Select the **Review + create** button at the bottom of the page.
+
+ ![At the bottom of the Create Web App screen, select 'Review + create'.](../../media/quickstart-php/azure-portal-create-app-service-review-create.png)   
+
+
+9. After validation runs, select the **Create** button at the bottom of the page.
+
+10. After deployment is complete, select **Go to resource**.
+
+### Set up continuous deployment
+
+This step will set up continuous deployment using GitHub actions.
+
+1. In the Azure portal, navigate to the app service.
+   
+2. From the navigation, select **Deployment Center**.
+
+![On the App Service resource, under the Deployment section of navigation, select Deployment Center.](../../media/quickstart-php/azure-portal-configure-app-service-deployment-center.png)  
+
+3. Under **Settings**, select a **Source**. For this quickstart, select `GitHub`.
+
+4. In the section under **GitHub**, select the following settings:
+    - Organization: Select your organization.
+    - Repository: Select `php-docs-hello-world`.
+    - Branch: Select `main`.
+
+5. Select **Save**.
+
+![Verify organization, repository, and branch settings. Then select Save from the actions bar.](../../media/quickstart-php/azure-portal-configure-app-service-github-integration.png)  
+
+> [!TIP]
```

Comment:
like the screenshots, tab this note block to make it line up with the numbered list.

Reply:
Updating images and all call outs (notes, tips, etc.) to indent as needed.